parse_opcodes

```diff
@@ -208,7 +218,8 @@ csrs = [
   (0x7a3, 'tdata3'),
   (0x7b0, 'dcsr'),
   (0x7b1, 'dpc'),
-  (0x7b2, 'dscratch'),
+  (0x7b2, 'dscratch0'),
```

Comment:
I feel like this change might cause some issues.  Make sure Spike still builds correctly.